### PR TITLE
subdomain updates

### PR DESCRIPTION
**Pull request by terakilobyte**
## Pull Request Info

This contains the necessary updates for spark subdomain changes.

[PR Reviewing Guidelines](https://github.com/mongodb/docs-spark-connector/blob/master/REVIEWING.md)

### Issue JIRA link:
n/a

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=623b38c20ab54725f188b961

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/spark-connector/docsworker-xlarge/subdomain-updates/

### Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Does it render on staging correctly?
- [ ] Are all the links working?
- [ ] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: config/intersphinx.yaml [hunk 1]
```diff
@@ -3,6 +3,6 @@ url: https://docs.python.org/2/
 path: python2.inv
 ---
 name: mongodb
-url: https://docs.mongodb.com/manual/
+url: https://mongodb.com/docs/manual/
```
Comment (biniona-mongodb) [inline config/intersphinx.yaml:6]: Issue: Could everything in the config directory except config/redirects be deleted? I believe this is appropriate as the repo is on next-gen publishing. 

Excerpt of [relevant blog post](https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=109644393):

>  Delete everything in the config/ directory except config/redirects , which is still used for redirects . Delete conf.py . Delete the Makefile . Shudder in delight at all the deleting.

File: structure.csv [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.
```
Comment (biniona-mongodb) [inline structure.csv:1]: Possible Issue: What is the purpose of this file? I can't find a wiki article or other docs repo that includes a comparable file. Should this be included in this PR?

**Review by biniona-mongodb — CHANGES_REQUESTED** [review]
Sorry for the annoying review. Totally understand why these edits were made, but after looking the the snooty next-gen migration guide it seems like these files should be deleted rather than updated.

File: snooty.toml [hunk 3]
```diff
@@ -15,4 +15,4 @@ scala-version-full = "2.12.13"
 [substitutions]
 copy = "unicode:: U+000A9"
 ent-build = "MongoDB Enterprise"
-hardlink = "http://docs.mongodb.com/docs-spark-connector"
+hardlink = "http://mongodb.com/docs/spark-connector/current"
```
[the remark this answers is not in the document]
Comment (terakilobyte) [reply snooty.toml:18]: I don't see any harm or benefit in removing it atm. I'll make sure to ask Zach about it

**Pull request closed without merging**